bin/hdfcoinc/pycbc_page_snrratehist

```diff
@@ -26,17 +28,41 @@ parser.add_argument('--output-file')
 parser.add_argument('--bin-size', type=float, default=0.1)
 parser.add_argument('--x-min', type=float, default=8.0)
 parser.add_argument('--trials-factor', type=int, default=1)
-parser.add_argument('--cumulative', action='store_true')
+parser.add_argument('--h-iter-inc-background', type=int, default=None,
+                    help='Indicate which inclusive background to plot '
+                         'with the foreground triggers if there were '
+                         'any hierarchical removals done. Choosing 0 '
+                         'indicates plotting the inclusive background '
+                         'prior to any hierarchical removals. Choosing 1 '
+                         'indicates the inclusive background after the '
+                         'loudest foreground trigger was removed. '
+                         '[default=None]')
```

Comment:
You need to explain what happens with the default... i.e., `Default (None) is to plot the background after all hierarchical removals were carried out.`